integration_tests/dbt_project.yml

```diff
@@ -8,7 +8,7 @@ vars:
   error_empty_seeds: true
   ## Use this variable to run the project with synthetic data loaded as seeds
   # otherwise set to false and set the input vars below
-  use_synthetic_data: false
```

Comment:
I believe we want to leave this default to false. That way synthetic data is not used by accident.

Reply:
good point! I agree.